addons/mass_mailing/models/mailing.py

```diff
@@ -837,6 +837,27 @@ def action_fetch_favorites(self, extra_domain=None):
 
         return values_list
 
+    def action_model_mass_mail(self, model_id, user, records):
+        """ Open a new mailing with the passed records as default target domain """
+        context = {
+            'default_reply_to_mode': 'new',
```

Comment:
Why ? Actually depends if model allows answers or not. There is already a compute for this field anyway, so don't think we have to force it here.